servers/rendering/renderer_rd/shaders/screen_space_reflection.glsl

```diff
@@ -175,17 +175,18 @@ void main() {
 	if (found) {
 		float margin_blend = 1.0;
 
-		vec2 margin = vec2((params.screen_size.x + params.screen_size.y) * 0.5 * 0.05); // make a uniform margin
-		if (any(bvec4(lessThan(pos, -margin), greaterThan(pos, params.screen_size + margin)))) {
-			// clip outside screen + margin
+		vec2 margin = vec2((params.screen_size.x + params.screen_size.y) * 0.05); // make a uniform margin
```

Comment:
Note that compared to the `3.x` code, I had to remove the half-resolution calculations for the margin to work properly.